doc/python/b64.md

```diff
@@ -0,0 +1,198 @@
+---
```

Comment:
How is this section used?

Reply:
This header in this section is copied from other examples.
To run this example you may run
```sh
jupyter notebook doc/python/b64.md
```